UITests/TrackingProtectionTests.swift

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Storage
+import EarlGrey
+@testable import Client
+
+class TrackingProtectionTests: KIFTestCase {
+    
+    private var webRoot: String!
+        
+    override func setUp() {
+        super.setUp()
+        webRoot = SimplePageServer.start()
+        BrowserUtils.dismissFirstRunUI()
+    }
+    
+    override func tearDown() {
+        BrowserUtils.clearPrivateData(tester: tester())
+        super.tearDown()
+    }
+    
+    func visitTrackingSite(blockerStatus: Bool) {
+        let url = "\(webRoot!)/tracking-protection-test.html"
+        EarlGrey.select(elementWithMatcher: grey_accessibilityID("url")).perform(grey_tap())
+        EarlGrey.select(elementWithMatcher: grey_accessibilityID("address")).perform(grey_replaceText(url))
+        EarlGrey.select(elementWithMatcher: grey_accessibilityID("address")).perform(grey_typeText("\n"))
+        
+        if blockerStatus {
+            EarlGrey.select(elementWithMatcher: grey_accessibilityLabel("image not loaded."))
+                .assert(grey_notNil())
+        } else {
+            EarlGrey.select(elementWithMatcher: grey_accessibilityLabel("image loaded."))
+            .assert(grey_notNil())
+        }
+        
+        EarlGrey.select(elementWithMatcher: grey_accessibilityLabel("OK"))
+            .inRoot(grey_kindOfClass(NSClassFromString("_UIAlertControllerActionView")!))
+            .assert(grey_enabled())
+            .perform((grey_tap()))
+        
+        BrowserUtils.resetToAboutHome(tester())
+    }
+    
+    func openTPSetting() {
+        // Check tracking protection is enabled on private tabs only in Settings
+        let menuAppeared = GREYCondition(name: "Wait the Settings dialog to appear", block: { () -> Bool in
```

Comment:
nit: `Wait for the Settings`

Reply:
nit to use trailing closure syntax:
```
let menuAppeared = GREYCondition(name: "Wait the for the Settings dialog to appear") { () -> Bool in
// do stuff
}
```

(can also leave out the `() -> Bool in` as closures that take no input args are allowed to skip that line and the return type is treated as implicit)